_includes/v21.2/sidebar-data/deploy.json

```diff
@@ -0,0 +1,416 @@
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Multi-region Capabilities",
+      "items": [
+        {
+          "title": "Overview",
+          "urls": [
+            "/${VERSION}/multiregion-overview.html"
+          ]
+        },
+        {
+          "title": "Choosing a Multi-region Configuration",
+          "urls": [
+            "/${VERSION}/choosing-a-multi-region-configuration.html"
+          ]
+        },
+        {
+          "title": "When to use ZONE vs. REGION Survival Goals",
+          "urls": [
+            "/${VERSION}/when-to-use-zone-vs-region-survival-goals.html"
+          ]
+        },
+        {
+          "title": "When to use REGIONAL vs. GLOBAL Tables",
+          "urls": [
+            "/${VERSION}/when-to-use-regional-vs-global-tables.html"
+          ]
+        },
+        {
+          "title": "Data Domiciling",
+          "urls": [
+            "/${VERSION}/data-domiciling.html"
+          ]
+        },
+        {
+          "title": "Migrate to Multi-region SQL",
+          "urls": [
+            "/${VERSION}/migrate-to-multiregion-sql.html"
+          ]
+        },
+        {
+          "title": "Topology Patterns",
+          "items": [
+            {
+              "title": "Overview",
+              "urls": [
+                "/${VERSION}/topology-patterns.html"
+              ]
+            },
+            {
+              "title": "Development",
+              "urls": [
+                "/${VERSION}/topology-development.html"
+              ]
+            },
+            {
+              "title": "Basic Production",
+              "urls": [
+                "/${VERSION}/topology-basic-production.html"
+              ]
+            },
+            {
+              "title": "Regional Tables",
+              "urls": [
+                "/${VERSION}/regional-tables.html"
+              ]
+            },
+            {
+              "title": "Global Tables",
+              "urls": [
+                "/${VERSION}/global-tables.html"
+              ]
+            },
+            {
+              "title": "Follower Reads",
+              "urls": [
+                "/${VERSION}/topology-follower-reads.html"
+              ]
+            },
+            {
+              "title": "Follow-the-Workload",
+              "urls": [
+                "/${VERSION}/topology-follow-the-workload.html"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Explore CockroachDB Features",
+      "items": [{
+          "title": "Replication & Rebalancing",
+          "urls": [
+            "/${VERSION}/demo-replication-and-rebalancing.html"
+          ]
+        },
+        {
+          "title": "Fault Tolerance & Recovery",
+          "urls": [
+            "/${VERSION}/demo-fault-tolerance-and-recovery.html"
+          ]
+        },
+        {
+          "title": "Multi-region Performance",
```

Comment:
Why is it Multi-region, when all other hyphenated titles are all initial caps?

Reply:
Fixed.